src/Logary/TemplateFormat.fs

```diff
@@ -0,0 +1,101 @@
+﻿[<AutoOpen>]
+module TemplateFormat
```

Comment:
Why not make this a single module, like so: `[<AutoOpen>] module internal Logary.LogaryCapturing`?